scripts/rest_client.py

```diff
@@ -156,6 +156,14 @@ def show_config(self):
         """
         return self._get("show_config")
 
+    def history(self, limit=None):
```

Comment:
This name seems quite inconsistent (sorry for not noticing before)...

How about we rename it to `trades` ? 
All other methods are named according to their API endpoint (`/trades` ...) - and i'd keep it at that.